provisioning/meta/dev.yml

```diff
@@ -0,0 +1,6 @@
+---
```

Comment:
i guess this should be in `playbooks`

imo we should choose, `provisioning` or `meta` as a tld. i use `meta` by convention, but don't really care too much. both seems... unnecessary :stuck_out_tongue: